protobuf/drand/control.proto

```diff
@@ -172,7 +172,7 @@ message BeaconStatus{
 }
 
 message ChainStoreStatus{
-    bool is_any_round = 1;
+    bool is_empty = 1;

```

Comment:
this will be a breaking proto change. it's on the control channel though so should generally be safe

Reply:
I know this is a breaking proto change, but nobody has used this feature yet. It was merged on a recent PR. That is why there is no problem on updating it. I think the new name makes more sense.